### PR TITLE
Feature/vault side bars

### PR DESCRIPTION
**Pull request by matthew-garrett**
- Migrates portfolio vault drawers to vaults feature shell
- Adds basis for new vault drawers
- Updates vault urls

### CODE REVIEW

File: packages/features/vault/src/all-strategies-view/all-strategies-view.tsx [hunk 1]
```diff
@@ -16,6 +17,9 @@ export const AllStrategyView = () => {
         capacityUsedPercentage,
         capacityRemaining,
       } = v;
+
```
Comment (jeffywu) [inline packages/features/vault/src/all-strategies-view/all-strategies-view.tsx:20]: Remove this

File: packages/features/vault/src/side-drawers/manage-vault/manage-vault.tsx [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+              display: 'flex',
+              alignItems: 'center',
+              width: '100%',
+              padding: theme.spacing(2.5),
+            }}
+          >
+            <H4
+              sx={{
+                flex: 1,
+                color: theme.palette.common.black,
```
Comment (jeffywu) [inline packages/features/vault/src/side-drawers/manage-vault/manage-vault.tsx:59]: is this color override required?

File: packages/features/vault/src/side-drawers/manage-vault/manage-vault.tsx [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+          <Link
+            to={link}
+            style={{
+              display: 'flex',
+              alignItems: 'center',
+              width: '100%',
+              padding: theme.spacing(2.5),
+            }}
+          >
+            <H4
```
Comment (jeffywu) [inline packages/features/vault/src/side-drawers/manage-vault/manage-vault.tsx:56]: The typography object (H4 here) takes a to= prop and will wrap the text in react-router-dom Link component. I'd use that instead here to reduce the amount of code and you can combine the sx props

**Review by jeffywu — APPROVED** [review]

File: packages/features/vault/src/side-drawers/manage-vault/manage-vault.tsx (hunk 3 above)
Comment (matthew-garrett) [reply packages/features/vault/src/side-drawers/manage-vault/manage-vault.tsx:56]: I was planning on coming back to fix these up later but I'll take care of them now.

**Pull request merged**